src/searchCountry.js

```diff
@@ -0,0 +1,99 @@
+import axios from "axios";
+
+let countryName = "nederland"
+
+async function specificCountryData() {
+    let countryData;
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${countryName}`);
+        countryData = result.data.map((data) => {
+            return data;
+        })
+    } catch (e) {
+        console.error(e.message);
+        console.log("massage" + e.message)
+        const element = document.getElementById("countryResult");
+        element.innerHTML = `
+                    <h1>Check je spelling!</h1>
+                    `
+    }
+    return countryData;
+}
+
+function countryData() {
+    specificCountryData().then(data => {
+        const {
+            flag,
+            name: countryName,
+            subregion,
+            population,
+            capital,
+        } = data[0]
+
+        let valuta = ""
+        if (data[0].currencies.length === 2) {
+            const [first, second] = data[0].currencies
+            const {name: valutaName1} = first
+            const {name: ValutaName2} = second
+            valuta = `${valutaName1} and ${ValutaName2}'s`
+        } else {
+            const [first] = data[0].currencies
+            const {name: valutaName} = first
+            valuta = `${valutaName}`
+        }
+
+        let language = ""
+        const [first, second, third] = data[0].languages
+        if (data[0].languages.length === 3) {
+            language = `They speak ${first.nativeName}, ${second.nativeName} and ${third.nativeName}`
+        }
+        if (data[0].languages.length === 2) {
+            language = `They speak ${first.nativeName} and ${second.nativeName}`
+        }
+        if (data[0].languages.length === 1) {
+            language = `They speak ${first.nativeName}.`
+        }
```

Comment:
Dit is niet echt onderhoudbaar als er meerdere languages en/of currencies bij komen. Wellicht dat je de array.join() functie kan gebruiken 😉